gui/freeadmin/forms.py

```diff
@@ -106,15 +107,18 @@ def _reroll(self):
             users = FreeNAS_Users(flags=FLAGS_DBINIT | FLAGS_CACHE_READ_USER)
         except:
             users = []
+        kwargs = {
+            'api_name': 'v1.0',
+            'resource_name': 'account/json_users'
+        }
         if len(users) > 500:
             if self.initial:
                 self.choices = ((self.initial, self.initial),)
-            kwargs = {}
             if len(self._exclude) > 0:
                 kwargs['exclude'] = ','.join(self._exclude)
```

Comment:
How is the exclude being handled? It used to be a url kwarg but it doesn't seem that exist anymore and it is a GET param? So it seems like this wont work?